api/tests/prescription/campaign/integration/application/api/campaigns-api_test.js

```diff
@@ -142,4 +143,65 @@ describe('Integration | Application | campaign-api', function () {
       ]);
     });
   });
+
+  describe('#deleteCampaigns', function () {
+    it('should delete campaigns and participations', async function () {
+      const admin = databaseBuilder.factory.buildUser();
+
+      const organizationId = databaseBuilder.factory.buildOrganization().id;
+      databaseBuilder.factory.buildPixAdminRole({ userId: admin.id, role: PIX_ADMIN.ROLES.SUPPORT });
+
+      const campaign = databaseBuilder.factory.buildCampaign({
```

Comment:
Nitpick: campaignId plutôt que campaign.id. car passage plus simple dans les appels à venir et homogénéité par rapport à organizationId